src/main/convert.js

```diff
@@ -163,8 +163,16 @@ export class ThriftFileConverter {
         )
       )
       .map(p => (p.indexOf('/') === -1 ? `./${p}` : p))
-      .map(relpath => `import * as ${path.basename(relpath)} from '${relpath}.js';`)
-      .join('\n');
+      .map(relpath => `import * as ${path.basename(relpath)} from '${relpath}.js';`);
 
+      if (this.isLongDefined()) {
+        generatedImports.push('import {default as Long} from \'long\';')
+      }
+      return generatedImports.join('\n');
+    }
   getImportAbsPaths = () => Object.keys(this.thrift.idls).map(p => path.resolve(p));
+
+  isLongDefined = () => {
+    return JSON.stringify(this.thriftAstDefinitions).indexOf('"js.type":"Long"') !== -1;
```

Comment:
I feel this is a hacky way of doing this but possibly more efficient and cleaner vs digging into the entire AST. Let me know how you feel about this.

Reply:
Thanks for this PR! I'd much prefer looking in the AST especially since that's how we add Long to the JS in the first place